.evergreen/run-tests.sh

```diff
@@ -139,7 +139,7 @@ if [ -n "$TEST_ENCRYPTION" ]; then
     export PYMONGOCRYPT_LIB
 
     # TODO: Test with 'pip install pymongocrypt'
-    git clone --branch master https://github.com/mongodb/libmongocrypt.git libmongocrypt_git
+    git clone --branch PYTHON-3285 https://github.com/ShaneHarvey/libmongocrypt.git libmongocrypt_git
```

Comment:
This needs to be removed once https://github.com/mongodb/libmongocrypt/pull/358 is merged.